### PR TITLE
Welcome card added to NoteContainer

### PR DESCRIPTION
**Pull request by trbachmann**
- Refactors NoteContainer to a class
- Adds a welcome message if there are no notes
screen shot 2019-02-19 at 4 03 52 pm

### CODE REVIEW

File: src/containers/NoteContainer/NoteContainer.js [hunk 1]
```diff
@@ -1,49 +1,70 @@
...
+      title: 'Welcome to Keeper',
+      listItems: [
+        { id: 'firstInstruction', description: 'Keep track of your projects', isComplete: false },
+        { id: 'secondInstruction', description: 'List out your tasks to track', isComplete: false },
+        { id: 'thirdInstruction', description: 'Click the plus sign above to get started', isComplete: false },
+      ],
+      color: 'white'
+    };
+    return <NoteCard {...welcomeMessage} key={welcomeMessage.id}/>
+  }
```
Comment (whitneyfirth) [inline src/containers/NoteContainer/NoteContainer.js:22]: Nice thinking on creating a unique welcomeMessage object! 🤩

File: src/containers/NoteContainer/NoteContainer.js [hunk 2]
```diff
@@ -1,49 +1,70 @@
...
+      784: 2,
+      528: 1
+    };
+    const disabledClass = isDisabled ? '--disabled' : '';
 
-  const disabledClass = isDisabled ? '--disabled' : '';
+    if (!notes.length) {
+      const welcome = this.addWelcomeNote();
+      cards.push(welcome);
+    }
```
Comment (whitneyfirth) [inline src/containers/NoteContainer/NoteContainer.js:43]: This is so easily readable, great job Tiffany!

**Review by whitneyfirth — APPROVED** [review]
You RULE!!!

**Review by dForDeveloper — APPROVED** [review]
Reviewed this in person. Looks good. 👍

**Pull request merged**